packages/commonwealth/test/integration/api/external/getTopics.spec.ts

```diff
@@ -18,6 +18,7 @@ describe('getTopics Tests', () => {
     const resp = await get('/api/topics', r);
 
     chai.assert.lengthOf(resp.result.topics, 2);
+    chai.assert.isNotNull(resp.result.topics.latest_activity);
```

Comment:
Shouldn't it be _per topic_?

Reply:
Yep, I was looking for notNull, but it was undefined. Damn javascript